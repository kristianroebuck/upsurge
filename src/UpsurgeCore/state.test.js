import getInitialState from './state';

describe('state', () => {
  it('should...', () => {
    const state = getInitialState();

    expect(state).toMatchSnapshot();
  });
});
