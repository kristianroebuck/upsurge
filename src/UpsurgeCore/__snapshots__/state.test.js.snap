// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`state should... 1`] = `
Object {
  "files": Array [],
}
`;
